src/main/java/mate/academy/spring/service/UserMapper.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.dto.UserResponseDto;
+import mate.academy.spring.model.User;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
does this component contain business logic?

Reply:
No, but this class into a service package